datafusion/physical-plan/src/aggregates/row_hash.rs

```diff
@@ -365,6 +459,36 @@ impl GroupedHashAggregateStream {
             merging_group_by: PhysicalGroupBy::new_single(agg_group_by.expr.clone()),
         };
 
+        // Skip aggregation is supported if:
+        // - aggregation mode is Partial
+        // - input is not ordered by GROUP BY expressions,
+        //   since Final mode expects unique group values as its input
```

Comment:
Hello @korowa , I have a question about this comment. 
I thought input is required no group ordering here is because we can emit groups early if we have group ordering. https://github.com/apache/datafusion/blob/dbc78902582f4a63fc26d7359018ae85d2051343/datafusion/physical-plan/src/aggregates/row_hash.rs#L645
But it confused me when I saw your comment. Is comment not updated or I misunderstood?

Reply:
Yes, this part of comment seems to be wrong and misleading, since ordering doesn't cause duplicates in the output of Final[Partitioned] -- I'll check out what is the root cause of test failures for the ordered inputs with enabled skip partial.

Reply:
Thank you for catching this.

Reply:
Thanks for reply.